subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -183,22 +183,28 @@ struct proc_ctx *ull_cp_priv_create_local_procedure(enum llcp_proc proc)
 	}
 
 	switch (ctx->proc) {
+#if defined(CONFIG_BT_CTLR_LE_PING)
 	case PROC_LE_PING:
```

Comment:
it may be a bit messy, but how about removing PROC_LE_PING from the llcp_proc enum in ull_llcp_internal.h?

Reply:
Well I actually thought about it, and decided against, as it makes no difference re. mem usage, for now at least. Then if we decide to optimize and table'ise stuff, then we can add this - at least so I thought

Reply:
it makes no difference on memory usage, but it will help ensure that all conditional compiles are in place

Reply:
I realize this. I just find that it messes up the code (and makes it less readable) for no big reason. And changing this calls for changing the remaining parts of local/remote that switches on these and this is bound to be optimized. So, can you accept the compromise to postpone this modification to the enum until we perform the optimization